Osoti12exercises.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>I love that you included an error message here! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>